packages/core/helper-plugin/package.json

```diff
@@ -48,7 +54,8 @@
     "rollup": "2.46.0",
     "rollup-plugin-node-polyfills": "0.2.1",
     "rollup-plugin-peer-deps-external": "2.2.4",
-    "rollup-plugin-postcss": "4.0.0"
+    "rollup-plugin-postcss": "4.0.0",
```

Comment:
Is this one necessary?

Reply:
Currently,  I think so... Until we clean the components of this package I thought it was best to keep the old rollup config.
I am thinking of switching to webpack to bundle the lib anyway.

Reply:
Sounds good, let's remove it on later iterations 👍🏻